lib/irc/IrcServer.js

```diff
@@ -31,6 +31,27 @@ IrcServer.prototype.getReadableName = function() {
     return '';
 }
 
+/**
+ * Returns whether the server is configured to wait some time
+ * (see this.config.getDisconnectPresenceTimeoutMs()) before parting
+ * a user that has `/quit` and also wait getDeboucePartsMs before parting
+ * a user that has disconnected due to a net-split.
+ * @return {bool} this.config.debounceParts.
+ */
+IrcServer.prototype.shouldDebouceParts = function() {
+    return this.config.debounceParts;
+}
+
+/**
+ * Get the number of ms to debounce before bridging a `/quit` to a leave
+ * during a detected net-split. If the user rejoins a channel before bridging
+ * the quit to a leave, the leave will not be sent.
+ * @return {integer}
+ */
+IrcServer.prototype.getDeboucePartsMs = function() {
+    return this.config.debouncePartsMs;
```

Comment:
Missing examples in `config.sample.yaml`. Missing defaults set in the default config. Should group together related variables `debouncePartsMs`, `debounceParts`, `getDisconnectPresenceTimeoutMs` in its own section like we do for other things.